cupy/core/carray.pxi

```diff
@@ -124,7 +124,7 @@ cpdef str _get_header_source():
 
 cpdef function.Module compile_with_cache(
         str source, tuple options=(), arch=None, cachd_dir=None,
-        prepend_cupy_headers=True):
+        prepend_cupy_headers=True, backend="nvrtc"):
```

Comment:
Please use `''` for string literals for consistency. Ditto. for other strings, including error message.

Reply:
done